pkg/analyze/analyze.go

```diff
@@ -150,8 +150,9 @@ func (a AlwaysGreenAnalyzer) Analyze(ctx context.Context, obj *status.Object) st
 // AnalyzerRegister is a registry of analyzers.
 // It allows to register new analyzers and ignored GroupKinds.
 type AnalyzerRegister struct {
-	analyzerInits []eval.AnalyzerInit
-	ignored       []schema.GroupKind
+	analyzerInits    []eval.AnalyzerInit
+	ignoredKinds     []schema.GroupKind
+	ignoredResources []schema.GroupResource
```

Comment:
Might be better just to settle down on one. If resources is the way to go, just how about just ditching the kinds, and replacing all the usage with resource alternative instead?

Reply:
:+1: yes if possible. I will try to look into it.

Reply:
I think this is pretty tricky to do, because this also touches the  `GroupKindMatcher` (https://github.com/iNecas/kube-health/blob/main/pkg/analyze/generic.go#L56) and the problem is that the k8s `ObjectReference` doesn't know anything about the resource (which is problematic for example here https://github.com/iNecas/kube-health/blob/main/pkg/eval/query.go#L331). I will think about it.

Reply:
An option could be to keep the `ignoredKinds` and add/expose some new function in the loader for the `groupResource` to `groupVersionKind` translation